docs/Installation.md

```diff
@@ -376,9 +384,6 @@ sudo -u postgres psql -f $(perl -MFile::ShareDir -le 'print File::ShareDir::dist
 
 #### 4.2.3 Instructions for SQLite (Debian)
 
-> **Note:** Zonemaster with SQLite backend is not yet considered stable and anyway
-> not meant for an installation with heavy load.

```

Comment:
I guess we should have the same statement -- or no statement -- for all OS. Should the statement that "Zonemaster with SQLite backend is not yet considered stable and anyway not meant for an installation 
with heavy load" be put somewhere else or not at all?

Reply:
Good point. I removed it because this release brings a stable SQLite support. But indeed the second part of the note is still relevant.
I'll revert this specific commit and make a new PR to update this note in all sections.